### PR TITLE
fix: Update Wallet and Generic Folder empty state to use lottie animations.

### PR DESCRIPTION
**Pull request by Krishna2323**
### Fixed Issues
$ https://github.com/Expensify/App/issues/48013
PROPOSAL: https://github.com/Expensify/App/issues/48013#issuecomment-2310685510

### Tests
1. Go to search page > Select a workspace with no expenses > Verify Folder Empty State Animation is shown
2. Select `Trips` from LHS (filters) > Verify Trips Empty State Animation is shown
3. From FAB select `Book travel` > Verify Trips Empty State Animation is shown
4. Open a workspace with no tags, categories 
5. Open policy tags page > Verify Folder Empty State Animation is shown
6. Open policy categories page > Verify Folder Empty State Animation is shown
7. Open report fields page  > Verify Folder Empty State Animation is shown
8. Click `Add field` > Select list as type
9. Go to list values > Verify Folder Empty State illustration is shown
10. Go to `settings/wallet` > Verify Bank Vault animation is shown

- [x] Verify that no errors appear in the JS console

### Offline tests
1. Go to search page > Select a workspace with no expenses > Verify Folder Empty State Animation is shown
2. Select `Trips` from LHS (filters) > Verify Trips Empty State Animation is shown
3. From FAB select `Book travel` > Verify Trips Empty State Animation is shown
4. Open a workspace with no tags, categories 
5. Open policy tags page > Verify Folder Empty State Animation is shown
6. Open policy categories page > Verify Folder Empty State Animation is shown
7. Open report fields page  > Verify Folder Empty State Animation is shown
8. Click `Add field` > Select list as type
9. Go to list values > Verify Folder Empty State illustration is shown
10. Go to `settings/wallet` > Verify Bank Vault animation is shown

### QA Steps
1. Go to search page > Select a workspace with no expenses > Verify Folder Empty State Animation is shown
2. Select `Trips` from LHS (filters) > Verify Trips Empty State Animation is shown
3. From FAB select `Book travel` > Verify Trips Empty State Animation is shown
4. Open a workspace with no tags, categories 
5. Open policy tags page > Verify Folder Empty State Animation is shown
6. Open policy categories page > Verify Folder Empty State Animation is shown
7. Open report fields page  > Verify Folder Empty State Animation is shown
8. Click `Add field` > Select list as type
9. Go to list values > Verify Folder Empty State illustration is shown
10. Go to `settings/wallet` > Verify Bank Vault animation is shown

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

</details>

https://github.com/user-attachments/assets/831db43b-c507-4e0c-a18b-492228933a71

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/user-attachments/assets/addb51cc-747c-47f7-9f0e-82533b2275da

</details>

<details>
<summary>iOS: Native</summary>

https://github.com/user-attachments/assets/70bb3fcc-1c70-4957-8432-ae202ffbb8bb

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/user-attachments/assets/4e46b83c-8429-424b-a6fb-d5e5ec6a3849

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/user-attachments/assets/97cc74b2-72ce-47e3-bbea-a47593dec727

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/user-attachments/assets/756a5670-57d0-4bc0-9aea-c64c0f4dd5dc

</details>

### CODE REVIEW

**Comment by dubielzyk-expensify** [comment]
Quick question for the @Expensify/design team. On desktop and we seem to be using `32px` padding on the card here, but in our empty state cards we go from `32px` to `20px` on mobile. Do we wanna do the same here. The padding around the button seem a tad big in this screenshot:

image

**Comment by shawnborton** [comment]
> but in our other empty state card component we go from 32px to 20px on mobile.

Can you show me an example of that? I don't feel too strongly here... the 32px keeps things fluffy but I can see where it might feel off given that the workspace cards and other places go from 32px to 20px.

**Comment by shawnborton** [comment]
@Krishna2323 everything is looking good to me, including button placement for categories! I'm going to follow up and get you a smaller version of the folder animation so that it feels more close to the older static version we had.

**Comment by dannymcclain** [comment]
I'd love to see the `20px` padding for small viewports/the RHP. I agree with Jon that the padding looks a _tad_ large.

**Comment by shawnborton** [comment]
Updated Lottie file is here: [GenericEmptystate-082724-1.json.zip](https://github.com/user-attachments/files/16763849/GenericEmptystate-082724-1.json.zip)

@roryabraham can you compress that for us when you get a chance? Thanks!

**Comment by shawnborton** [comment]
Sounds good, keep us posted!

**Comment by shawnborton** [comment]
@Krishna2323 would it be easy for you to change the padding of this area on mobile:
CleanShot 2024-08-27 at 12 58 24@2x

To be 20px?
CleanShot 2024-08-27 at 12 59 02@2x

**Comment by roryabraham** [comment]
Updated animation GenericEmptyState animation: [GenericEmptyState.lottie.zip](https://github.com/user-attachments/files/16787583/GenericEmptyState.lottie.zip)

**Comment by dubielzyk-expensify** [comment]
> Can you show me an example of that? I don't feel too strongly here... the 32px keeps things fluffy but I can see where it might feel off given that the workspace cards and other places go from 32px to 20px.

@shawnborton :

CleanShot 2024-08-29 at 09 52 56@2x

**Comment by dubielzyk-expensify** [comment]
This screen looks wrong to me and it's duplicating the animation which we don't want. 
CleanShot 2024-08-29 at 14 17 10@2x

Maybe it would play if we don't have the animation in the Right Hand Panel as well? cc @Expensify/design

**Comment by hungvu193** [comment]
> . When the RHP opens, the animation in the background stops. I’ve been trying to find a solution for that, but I haven't found any clues yet. @hungvu193, do you have any ideas about this?



It came from https://github.com/Expensify/App/pull/47960, we're hiding lottie view when navigation is not focused.

 https://github.com/user-attachments/assets/3196086a-ab5c-455f-9c94-7aa6241f0cd9

**Comment by hungvu193** [comment]
It was aimed to fix https://github.com/Expensify/App/issues/47273. So I think that wont be an issue then? Would love to hear from design team.

**Comment by shawnborton** [comment]
> This screen looks wrong to me and it's duplicating the animation which we don't want.

@dubielzyk-expensify why do you think it looks wrong? I guess it's technically an empty list, so we're showing the empty state we use for empty lists there... I don't feel strongly though, I could see using a lighter empty state for this one.

**Comment by shawnborton** [comment]
I think ideally the animation would still play in the background when the RHP is open though... so I would love to solve that if possible.

**Comment by hungvu193** [comment]
> I think ideally the animation would still play in the background when the RHP is open though... so I would love to solve that if possible. 

Can you please comment in the original issue that I mentioned above?. It's not related to this PR 😄

**Comment by shawnborton** [comment]
I can totally get down with that, I'm happy to use a static illustration there so we don't duplicate the animation.

**Comment by dubielzyk-expensify** [comment]
> @dubielzyk-expensify why do you think it looks wrong? I guess it's technically an empty list, so we're showing the empty state we use for empty lists there... I don't feel strongly though, I could see using a lighter empty state for this one.

Yeah that was my thinking, but if you and @dannymcclain think it looks alright, then that's good to me

**Comment by shawnborton** [comment]
> could you please provide the static illustration if we need to replace the animation?

The static illustration already exists, it's basically the current implementation on main. 

> Yeah that was my thinking, but if you and @dannymcclain think it looks alright, then that's good to me

No strong feelings here. I guess we could just keep what's there (the list empty state + static illustration) and follow up to make it lighter outside of this PR, since this PR is mainly about getting the animated empty state in other places.

**Comment by dannymcclain** [comment]
+1 to what Shawn said.

The folder illo is here: `App/assets/images/product-illustrations/folder-with-papers.svg` but I agree the focus of this PR is adding the animations where we want them.

**Comment by dannymcclain** [comment]
The animation/illustration is looking good to me.

>The only thing that does not seems correct to me is using the animation for category selector RHP

I'm not quite understanding this—can you share a screenshot/video so I can see what you mean?

**Comment by shawnborton** [comment]
I didn't even realize that page existed. I kind of think the empty state here doesn't really represent the category selection page very well - in that the category selection page isn't really a table view, it's a list selection page. So I would be down to just use a super generic/clean "no categories here" empty state, something like we do when you go to create a room and we have no workspaces available: 
CleanShot 2024-09-03 at 09 42 45@2x

**Comment by shawnborton** [comment]
Hmm can you show us the actual, real flow then so we can better evaluate this?

**Comment by shawnborton** [comment]
Got it, thanks for explaining. I mean I don't feel too strongly here, I think what you are showing [above](https://github.com/Expensify/App/pull/48040#issuecomment-2326075156) actually looks fine?

**Comment by dannymcclain** [comment]
>I'm not sure why policies without a category are not shown when categorizing a track expense. I guess that’s a bug.

This was an intentional choice, not a bug as far as I recall.

I don't feel super super strongly, but I do agree with Shawn that the regular empty state in that situation (empty select list) makes more sense. If we want to go with what you're showing, at the very least we'll need to update the text.

**Comment by shawnborton** [comment]
I think it makes the most sense to mimic the empty state I shared [here](https://github.com/Expensify/App/pull/48040#issuecomment-2325818452), mostly since this is a list selection screen and not a table view. If you are cool making those changes, that would be great.

**Comment by hungvu193** [comment]
> Thanks, I will update that page and share the screenshots very soon.
Only screenshots are left right?

**Comment by shawnborton** [comment]
That feels pretty good to me! Let's get this one into final review then.

**Comment by hungvu193** [comment]
Friendly bump @Krishna2323, let's wrap it up.

**Comment by shawnborton** [comment]
Gonna run a test build but so far the screen recording looks good to me!

File: src/styles/index.ts
Comment (hungvu193) [inline src/styles/index.ts:4739]: May I know why do you remove the width here?

File: src/components/EmptyStateComponent/types.ts
Comment (hungvu193) [inline src/components/EmptyStateComponent/types.ts:19]: Why do we need this props? I don't think I saw it's being used anywhere?

**Comment by shawnborton** [comment]
Looks like the empty state modal portion is too tall - it should only be 220px:
CleanShot 2024-09-10 at 09 56 10@2x

**Comment by shawnborton** [comment]
Wallet empty state height looks correct though:
CleanShot 2024-09-10 at 09 58 00@2x

**Comment by hungvu193** [comment]
on iPhone 15 simulator, the lottie view of EmptySearchView doesn't look correct.


https://github.com/user-attachments/assets/aecb6fbf-2e0c-4bba-8e85-224381f8273c

**Comment by shawnborton** [comment]
Ooh yeah good catch there too.

**Comment by shawnborton** [comment]
The trips empty state is too tall as well, again this one should be 220px:
CleanShot 2024-09-10 at 11 23 57@2x

I think we'd need to shrink down the animation - let me know if you need a new version in order to accomplish that or if we can manually adjust the width/height of it.

**Comment by hungvu193** [comment]
I can also reproduce above issue on Pixel 6

https://github.com/user-attachments/assets/6a9bab26-441f-4d4c-a00b-23da01ecde92

**Comment by shawnborton** [comment]
Hmm can you tell me what the dimensions are of the animation itself? Maybe we need to get you a new version that has a different size.

**Comment by shawnborton** [comment]
I'm going to ask for a new version, stay tuned.

**Comment by c3024** [comment]
I guess these [tags](https://github.com/Expensify/App/pull/48040#issuecomment-2342847140) must be for @hungvu193 .

**Comment by shawnborton** [comment]
Here is an updated file to try for the folder animation: [GenericEmptyState-400x220-091124-kjs-1.json.zip](https://github.com/user-attachments/files/16973462/GenericEmptyState-400x220-091124-kjs-1.json.zip)

@roryabraham do you mind also running this one through the lottie compressor?

**Comment by roryabraham** [comment]
[GenericEmptyState.lottie.zip](https://github.com/user-attachments/files/16982870/GenericEmptyState.lottie.zip)

**Comment by shawnborton** [comment]
@Krishna2323 nice, you have the new asset now - let's try to get this one over the finish line!

**Comment by shawnborton** [comment]
Looks better but I'm not sure that the animations should resize like that. Can we just keep them vertically and horizontally centered even when the screens get smaller?

**Comment by hungvu193** [comment]
Any updates @Krishna2323?

**Comment by dannymcclain** [comment]
I think that's looking pretty good to me! Will wait for Shawn's feedback too.

**Comment by shawnborton** [comment]
That does feel better, yup.

**Comment by shawnborton** [comment]
Gonna run a test build but let's get this into final review!

**Comment by shawnborton** [comment]
Hmm I'm definitely not trying to be difficult, but in this case here I would still expect the animation to be 220px tall - it would just be horizontally centered within the available background area. Instead it looks like we are shrinking based on the 400px width that it has:
CleanShot 2024-09-18 at 23 55 50@2x

Perhaps it's a simple fix? Take a look at this:


https://github.com/user-attachments/assets/8f4c8609-c201-4dde-99c9-85495cbeac96

File: src/styles/index.ts
Comment (hungvu193) [inline src/styles/index.ts:5173]: @Krishna2323 Friendly reminder that you need to make sure you test this one on smaller device like iPhone8

Comment (hungvu193) [reply src/styles/index.ts:5179]: Thanks man! Appreciate it, I'll give it a final review today!

**Comment by hungvu193** [comment]
I'm blocked from testing because of this bug

https://github.com/user-attachments/assets/bddbbf7b-ef31-4282-91b1-ceae0f8b44c7

**Comment by hungvu193** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>






https://github.com/user-attachments/assets/8df02369-1619-472c-a9d7-f79df2ada4fb


</details>

<details>
<summary>Android: mWeb Chrome</summary>




https://github.com/user-attachments/assets/91084b83-f216-444a-a72c-9c0605b3f12d

</details>

<details>
<summary>iOS: Native</summary>



https://github.com/user-attachments/assets/07afbc27-d37d-47f2-a9ec-1dc0c6c76d5e

</details>

<details>
<summary>iOS: mWeb Safari</summary>



https://github.com/user-attachments/assets/90fef056-7a3b-4990-b684-28dc941733db

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



https://github.com/user-attachments/assets/35da5904-5f24-4c7b-bb2a-8ab3c2421d60

</details>

<details>
<summary>MacOS: Desktop</summary>




https://github.com/user-attachments/assets/8a2fe916-05fb-4bee-9da2-574289bea966

</details>

**Review by hungvu193 — APPROVED** [review]

File: src/components/EmptyStateComponent/index.tsx
Comment (roryabraham) [inline src/components/EmptyStateComponent/index.tsx:33]: NAB because it seems like all the styles are right in the screen recordings, but I think we might be able to use `shouldUseNarrowLayout` here, and generally that should be the default go-to instead of `isSmallScreenWidth`

**Review by roryabraham — APPROVED** [review]

**Comment by MonilBhavsar** [comment]
This PR seem to have caused a regression https://github.com/Expensify/App/issues/49744

**Comment by hungvu193** [comment]
> This PR seem to have caused a regression #49744

This PR only updated the lottie animation files so it won't cause that regression. Ideally, the regression will be from one of the PR that updated LottieView recently.

**Pull request merged**
